.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Set up uv
+        run: curl -LsSf https://astral.sh/uv/install.sh | sh
+
+      - name: Restore uv cache
+        uses: actions/cache@v4
+        with:
+          path: /tmp/.uv-cache
+          key: uv-${{ runner.os }}-${{ hashFiles('uv.lock') }}
+          restore-keys: |
+            uv-${{ runner.os }}-${{ hashFiles('uv.lock') }}
+            uv-${{ runner.os }}
+
+      - name: Install Postgresql
+        uses: ikalnytskyi/action-setup-postgres@v6
+        with:
+          username: postgres
+        id: postgres
+
+      - name: Install dependencies
+        run: |
+          uv pip install --system -e .[dev]
+
+      - name: Test with pytest
+        run: |
+          pytest -s
+        env:
+          PGSERVICE: ${{ steps.postgres.outputs.service-name }}
```

Comment:
I'm surprised the tests just worked with these additions.  I can finally delete the outdated `ghtests` branch.

Reply:
Yes, I was also surprised that this was fairly simple. I also had to change the `DatabaseJanitor` a bit since it has changed in newer versions of `pytest_postgresql` but nothing too dramatic.